app/controllers/application_controller.rb

```diff
@@ -26,6 +26,7 @@ def append_info_to_payload(payload)
       payload[:user_organisation_slug] = current_user.organisation_slug
     end
     payload[:request_id] = request.request_id
+    payload[:remote_ip] = request.remote_ip
```

Comment:
Just approved it but now I'm wondering why we call it `payload.remote_ip` here and then `user_ip` below, can it be consistent?